README.md

```diff
@@ -20,9 +20,43 @@ Or install it yourself as:
 
     $ gem install defra_ruby_aws
 
+## Configuration
+
+Add a new bucket with:
+
+```
+# config/initializers/defra-ruby-aws.rb
+require "defra-ruby-aws"
+
+DefraRuby::Aws.configure do |config|
+  config.buckets = [{
+    # bucket's name, required
+    name: "defra-ruby-aws",
+    # AWS bucket access credentials, required
+    credentials: {
+      access_key_id: "ACCES_KEY_ID",
```

Comment:
```suggestion
      access_key_id: "ACCESS_KEY_ID",
```

Typo :) Are these the names of env vars? If so I think we should include that in the comment.